docs/develop/add-ins-with-angular2.md

```diff
@@ -73,24 +73,25 @@ The [displayDialogAsync](http://dev.office.com/reference/add-ins/shared/officeui
 It is important to remember, if you pass a route, that the dialog box creates a new window with its own execution context. Your base page and all its initialization and bootstrapping code run again in this new context, and any variables are set to their initial values in the dialog box. So this technique launches a second instance of your single page application in the dialog box. Code that changes variables in the dialog box does not change the task pane version of the same variables. Similarly, the dialog box has its own session storage, which is not accessible from code in the task pane.  
 
 
-## Forcing an update of the DOM
+## Trigger the UI update
 
-In any Angular application, notifications to update the DOM occasionally do not fire. The framework provides a `tick()` method on the `ApplicationRef` object that will force an update. The following code is an example.
+In an Angular app, the UI sometimes won't update. This is because that part of code runs out of Angular zone. The solution is putting that part of code in the zone. The following codes shows the way:
```

Comment:
Recast the second sentence as "This is because that part of the code runs in the Zone in Angular. The solution is to put the UI code in the Zone, as shown in the following example."